.github/workflows/ci-unix-static-sanitized.yml

```diff
@@ -107,6 +107,8 @@ jobs:
           CXXFLAGS: ${{ env.CI_CXXFLAGS }}
           LDFLAGS: ${{ env.CI_LDFLAGS }}
           LD_LIBRARY_PATH: ${{ env.CI_LD_LIBRARY_PATH }}
+          C_LD: clang
+          CXX_LD: clang
```

Comment:
It seems better to use the C++ compiler (clang++) as the linker so that it links with the C++ runtime code.

Reply:
done